R/utils.R

```diff
@@ -0,0 +1,73 @@
+#' Creates `AVAL` from `AVALC` by Calling User Function
+#'
+#' Create `AVAL` from `AVALC` by calling a function provided by the user. If
+#' calling the function fails, the error is catched and a helpful error message
+#' provided.
+#'
+#' @param dataset Input dataset
+#'
+#'   The variable `AVALC` is expected.
+#'
+#'   *Permitted Values*: a dataframe
+#'
+#' @param aval_fun Function returning the `AVALC` values
+#'
+#'   The specified function must expect one argument expecting a character
+#'   vector and must return a numeric vector.
+#'
+#'   *Permitted Values*: a function
+#'
+#' @details The new variable `AVAL` is set to `aval_fun(AVALC)`.
+#'
+#' @author Stefan Bundfuss
+#'
+#' @returns The input dataset with `AVAL` added
+#'
+#' @keywords utils_catch
+#' @family utils_catch
```

Comment:
utils_catch should also be added to man/roxygen/meta.R

Reply:
Added